src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -602,7 +671,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
-        Private ReadOnly Property IBinaryKind As BinaryOperationKind Implements IBinaryOperatorExpression.BinaryKind
```

Comment:
I don't understand the convention. @johnhanby why are these named this way?

On Jan 11, 2016 1:20 PM, Gen Lu <notifications@github.com> wrote:

In src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb<https://github.com/dotnet/roslyn/pull/7352#discussion_r49356698>:

> @@ -602,7 +671,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
>              End Get
>          End Property
>
> -        Private ReadOnly Property IBinaryKind As BinaryOperationKind Implements IBinaryOperatorExpression.BinaryKind


It's a naming convention used through out IOperation, I didn't change any of it.

-
Reply to this email directly or view it on GitHub<https://github.com/dotnet/roslyn/pull/7352/files#r49356698>.